app/views/steps/other_parties/address_details/edit.html.erb

```diff
@@ -7,7 +7,15 @@
     <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading', name: @form_object.record.full_name %></h1>
 
     <%= step_form @form_object do |f| %>
-      <%= f.text_area :address, size: '40x4', class: 'form-control-3-4' %>
+      <% if f.object.split_address? %>
+          <%= f.text_field :address_line_1 %>
```

Comment:
Sorry to be that guy LOL 🤣 indentation is off (I think it is also off in the other address details views and this is a copy&paste).